python/ray/_private/ray_logging/filters.py

```diff
@@ -5,6 +5,9 @@
 
 class CoreContextFilter(logging.Filter):
     def filter(self, record):
+        timestamp_ns = int(record.created * 1e9)
+        setattr(record, LogKey.TIMESTAMP_NS.value, timestamp_ns)
```

Comment:
We need to see how this works with `_setup_log_record_factory` in `python/ray/_private/ray_logging/logging_config.py` cc @kevin85421

Reply:
sg, added a test case for it. The `timestamp_ns` still works as expected